.github/workflows/pr.yml

```diff
@@ -113,13 +113,17 @@ jobs:
             sed -e 's/3.5 //g')
           for version in $PY_VERS; do
             ansible-test sanity --verbose --docker --python $version --color --coverage --failure-ok
+            ansible-test units --verbose --docker --python $version --color --coverage || :
```

Comment:
Is there a reason why the `--failure-ok` may not be enough here?

It is changed here in the coming `branch` but is not changed in the `main` branch.

Reply:
Good catch! Yes, there is a reason: it doesn't work the units sub-command.

Reply:
that's also there on the `main` branch...

Reply:
Nevermind, I did look it wrong. All good :+1: